src/env.cpp

```diff
@@ -50,6 +57,9 @@ extern char **environ;
 
 bool g_use_posix_spawn = false;  // will usually be set to true
 
+/// Does the terminal have the "eat_newline_glitch".
+int term_has_xn = 0;
```

Comment:
Just a consideration if you're coming from C where bool is a C99 thing to be regarded with suspicion. Either int or bool are fine.